config/component_metadata.yaml

```diff
@@ -0,0 +1,22 @@
+releases:
+  - name: KServe
+    version: v0.14.0
+    repoUrl: https://github.com/kserve/kserve/
+  - name: ModelMesh Serving
+    version: v0.12.0-rc0
+    repoUrl: https://github.com/kserve/modelmesh-serving
+  - name: vLLM
+    version: v0.6.4.post1
```

Comment:
This doesn't seems right `0.6.4.post2.dev115+g0ca5ce7b.d20241204` is the correct version

Reply:
Can we add a comment saying that this corresponds to the latest upstream tag merged into opendatahub-io/vllm?

Reply:
For example, right now we're at v0.6.4.post1 (https://github.com/opendatahub-io/vllm/releases/tag/v0.6.4.post1) 
other tags are going to be available here:
https://github.com/opendatahub-io/vllm/tags

Reply:
@dtrifiro @tarukumar I will let the feature lead know that we would want to include a little note in the UI saying that all components have extra commits on top of the upstream community release, so customers/users know that our images, while they include the upstream community runtime up to x.y.z release, we have additional commits on top of it.

Reply:
But I wouldn't include this comment here, since it is pertinent to all components (not just model serving).

Reply:
I think that leaving a note here aswell would do no harm, so whoever is updating this doesn't forget, but I'm fine either way